src/contacts/ConsoleReader.java

```diff
@@ -0,0 +1,74 @@
+package contacts;
+
+import contacts.service.Validator;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.regex.Pattern;
+
+public class ConsoleReader {
+    private static final BufferedReader READER = new BufferedReader(new InputStreamReader(System.in));
+
+    public static String getStringFromConsole(String message) {
+        System.out.println(message);
+        String value;
+        try {
+            value = READER.readLine();
+        } catch (IOException exception) {
```

Comment:
Please move this part to the private method too

Reply:
Done